.github/CODE_REVIEW_GUIDELINES.md

```diff
@@ -13,13 +13,12 @@
 - The scope of PR should be simple, unique and well-defined. PR should not contain unrelated changes
 - Approve PR only if you are sure about the scope
 - Be respectful and reply asap
-- Do not merge PR without approval
-- Avoid spending too much time on trivial changes
+- Avoid spending time on trivial changes
```

Comment:
Top much esa right; I think some minor style changes quickly add up

Reply:
Didn't get you

Reply:
edited